float64 x_b_dot
float64 y_b_dot
float64 z_b_dot
float64 theta_b_dot
